EIPS/eip-5380.md

```diff
@@ -0,0 +1,181 @@
+---
+eip: 5380
+title: EIP-4907 Alternative Design
+description: Add a time-limited role with restricted permissions to EIP-721 tokens.
+author: Anders (@0xanders), Lance (@LanceSnow), Shrug <shrug@emojidao.org>, Tim Daubenschütz (@TimDaub)
+discussions-to: https://ethereum-magicians.org/t/idea-erc-721-user-and-expires-extension/8572
+status: Final
```

Comment:
```suggestion
status: Draft
```
EIPs all must go through the process, no special treatment for any reason (too fuzzy, too much contention).  Note, you can proceed through all the steps except last call as fast as you can get editor approval/merge.